### PR TITLE
Session time is incorrect due to Time Zone change.

### LINKED ISSUE #322: Session time is incorrect due to Time Zone change.
## Overview (Required)
The reproduction procedure is as follows.

1) Visit Settings app.
1-1) Data & time > Set Time zone to London. (GMT+00:00)
1-2) Perform CLEAR DATA of droidkaigi2018 app.
2) Launch droidkaigi2018 app. (Sessions will be displayed soon.)
3) Finish app. (Back button)
4) Visit Settings app.
4-1) Data & time > Set Time zone to Tokyo. (GMT+09:00)
5) Launch droidkaigi2018 app. (The time of the displayed session is incorrect.)

sessiontimeincorrect

As an impression that I read the code, the following code has doubts.

LocalDateTimeAdapter.kt
```kotlin
private val FORMATTER: DateTimeFormatter =
        DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss")
fun parseDateString(dateString: String?): LocalDateTime =
        LocalDateTime.parse(dateString, FORMATTER)
```
I guess that the time obtained from the server is based on Japan time. 
However, the application is parsing according to current Time Zone set for the terminal?

## Links
-

### PR DESCRIPTION
**Pull request by nukka123**
## Issue
- close #322

## Overview (Required)
- Specify JST for Timezone concerning LocalDateTime conversion.

## Links
-

## Screenshot
Before | After
:--: | :--:
faild | fixed

### CODE REVIEW

**Comment by takahirom** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/Converters.kt [hunk 1]
```diff
@@ -13,11 +13,12 @@ object Converters {
...
-        return LocalDateTime.ofInstant(Instant.ofEpochSecond(value), ZoneId.systemDefault())
+        return LocalDateTime.ofInstant(
+                Instant.ofEpochSecond(value), ZoneId.of("JST", ZoneId.SHORT_IDS))
     }
 
     @JvmStatic
     @TypeConverter
     fun dateToTimestamp(date: LocalDateTime?): Long? =
-            date?.toEpochSecond(ZoneOffset.ofHours(9))
+            date?.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS))!!.toEpochSecond()
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/Converters.kt]: Probably you can do this 👍 

```
date!!.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS)).toEpochSecond()
```

**Comment by takahirom** [comment]
Many thanks 🙇 
I don't have time zone knowledge. Thanks 👍

File: app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/Converters.kt (hunk 1 above)
Comment (nukka123) [reply app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/Converters.kt]: I am kotlin beginner . . .

The suggestion is that an exception will occur if date is null.
If you follow the original code, how about the following code?
```
date?.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS))?.toEpochSecond()
```
Or should an exception occur?

**Comment by konifar** [comment]
LGTM!

**Pull request merged**